sphinx_gallery/tests/tinybuild/doc/Makefile

```diff
@@ -10,7 +10,7 @@ clean:
 	rm -rf gen_modules/
 
 html:
-	sphinx-build -b html -d _build/doctrees $(SPHINXOPTS) . _build/html
+	sphinx-build -b html -v -d _build/doctrees $(SPHINXOPTS) . _build/html
```

Comment:
I'd rather move the `-v` to SPHINXOPTS at the top

Reply:
Sorry, I didn't mean to commit that in the first place.